src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/court/CourtEmails.kt

```diff
@@ -583,3 +583,27 @@ class TransferredCourtBookingPrisonNoCourtEmail(
     addPersonalisation("postAppointmentInfo", postAppointmentInfo ?: "Not required")
   }
 }
+
+class CourtHearingLinkReminderEmail(
```

Comment:
I think this can extend `CourtEmail`, avoids a lot of the `init { ... }` duplication.

The only thing you would then need to include is `init { addPersonalisation("bookingId", bookingId) }`